src/test/java/dk/cngroup/lentils/service/ContactServiceTest.java

```diff
@@ -0,0 +1,88 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.Contact;
+import dk.cngroup.lentils.entity.FinalPlace;
+import dk.cngroup.lentils.exception.MoreContactsException;
+import dk.cngroup.lentils.exception.MoreFinalPlacesException;
+import dk.cngroup.lentils.repository.ContactRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ContactServiceTest {
+
+    @InjectMocks
+    ContactService service;
+
+    @Mock
+    ContactRepository repository;
+
+    private ObjectGenerator generator = new ObjectGenerator();
+
+    @Test
+    public void addNewContactTest() {
+        Contact contact = createNewContact();
+        when(repository.save(contact)).thenReturn(contact);
+
+        assertNotNull(contact);
```

Comment:
Tohle nevim, jestli je dobry test.
Tady vlastne testujes, jestli ti ta tvoje metoda v testovaci tride vytvari objekt.
Pak nastavit, aby repository.save vratila objekt a pak vlastne testuje jestli funguje ten Mock objekt. To asi nechces.

Reply:
Navic stejne volani when je uz ve funkci createnewContact jestli se divam spravne.

Reply:
mas pravdu, zrusila jsem ho, a upravila jsem createnewContact